webcompat/templates/issue/issue-labels-sub.jst

```diff
@@ -1,8 +1,7 @@
 <script type="text/template">
 <% _.each(labels, function(label) { %>
-  <span class="wc-Label wc-Label--badge js-Label"
-        style="background-color:#<%=label.color%>">
+  <span class="label label-<%= label.attributes[3].value %> js-Label">
```

Comment:
What is `label.attributes[3].value` all about?

Reply:
We use `label.remoteName` as our identifier for the class, but just pass on the `label.name` here. As this refactor is about markup and changing as less JS as possible, this seemed as the easiest solution.

Any other ideas?